### DISCUSSION TITLE
Changes in php.ini are not preserved

### DISCUSSION THREAD

**Discussion by UnSautEnAvant** [leokhoa/laragon#602] [Q&A]
Hello,

I'm trying to use MailHog with Laragon, following this tuto : https://pen-y-fan.github.io/2023/01/04/how-to-set-up-laragon-on-a-new-windows-computer-part-5/#further-information

I can access to the page http://mailhog:8025/ but when I send an email, it does not appear. So I searched for a problem and I think I found it.

I'm supposed to have this in php.ini : sendmail_path = "Z:/laragon/bin/mailhog/MailHog.exe sendmail"

But when I restart Laragon, the file changes on his own to : sendmail_path ="Z:/laragon/bin/sendmail/sendmail.exe"

I guess the problem comes from this, but I can't find a way to keep the line I've written.

Could you help me please ?

**Comment by UnSautEnAvant** [comment]
So, I edited the php.ini file and put it in "Read only" and it seems to resolve the problem... Maybe not the best way to do it, if someone as a better option...

**Reply by olexanderoff** [reply]
Just remove or comment out the sendmail_path line. It is enough to specify SMTP and smtp_port.

**Comment by josealgardataborges** [comment]
Couldn't reproduce this on my laragon, maybe you changed PHP versions in between setting up the *php.ini* file? 

Please, remember that each version of PHP where you need to use mailhog, you have to edit php.ini to add that line and comment the old one.

Also, don't forget to uncheck the LARAGON sendmail option.


![image](https://github.com/leokhoa/laragon/assets/8929655/a0472ec2-cd3d-45d7-be65-6d2f6c881395)
